modules/apps/frontend-theme/frontend-theme-dialect-style-guide-sample-web/src/main/resources/META-INF/resources/js/guides/TableGuide.js

```diff
@@ -0,0 +1,458 @@
+				{DENSITY.map((item) => (
+					<TokenItem
+						key={item.className}
+						label={item.className}
+						size="large"
+					>
+						<Table
+							className={item.className}
+							hover={item.hover}
+							isResponsive={item.responsive}
+							text="Spain"
+						/>
+					</TokenItem>
+				))}
+			</TokenGroup>
+
+			<TokenGroup
+				group="utility-classes"
+				title={Liferay.Language.get('utility-classes')}
+			>
+				<TokenItem label="table-striped" size="large">
+					<Table
+						className="table-striped"
+						hover={false}
+						isResponsive={false}
+						text="Spain"
+					/>
+				</TokenItem>
+
+				{UTILITY_CLASSES.map((item) => (
+					<TokenItem
+						key={item.className}
+						label={item.className}
+						size="large"
+					>
+						<Table
+							className={item.className}
+							hover={item.hover}
+							isResponsive={item.responsive}
+							text={BIG_TEXT}
+						/>
+					</TokenItem>
+				))}
+
+				<TokenItem label="table-cell-expand" size="large">
+					<Table
+						cellExpanded
+						hover={false}
+						isResponsive={false}
+						text={BIG_TEXT}
+					/>
+				</TokenItem>
+
+				<TokenItem label="table-img" size="large">
+					<ClayTable hover={false} responsive={false}>
+						<ClayTable.Head>
+							<ClayTable.Row>
+								<ClayTable.Cell headingCell>
+									Object
+								</ClayTable.Cell>
+
+								<ClayTable.Cell headingCell>
+									Size
+								</ClayTable.Cell>
+
+								<ClayTable.Cell headingCell>
+									Image
+								</ClayTable.Cell>
+							</ClayTable.Row>
+						</ClayTable.Head>
+
+						<ClayTable.Body>
+							<ClayTable.Row>
+								<ClayTable.Cell headingTitle>#1</ClayTable.Cell>
+
+								<ClayTable.Cell>60x30</ClayTable.Cell>
+
+								<ClayTable.Cell expanded>
+									<img
+										className="table-img"
+										src="https://via.placeholder.com/60x30"
+									/>
+								</ClayTable.Cell>
+							</ClayTable.Row>
+
+							<ClayTable.Row>
+								<ClayTable.Cell headingTitle>#2</ClayTable.Cell>
+
+								<ClayTable.Cell>60x120</ClayTable.Cell>
+
+								<ClayTable.Cell expanded>
+									<img
+										className="table-img"
+										src="https://via.placeholder.com/60x120"
+									/>
+								</ClayTable.Cell>
+							</ClayTable.Row>
+
+							<ClayTable.Row>
+								<ClayTable.Cell headingTitle>#3</ClayTable.Cell>
+
+								<ClayTable.Cell>100x100</ClayTable.Cell>
+
+								<ClayTable.Cell expanded>
+									<img
+										className="table-img"
+										src="https://via.placeholder.com/100x100"
+									/>
+								</ClayTable.Cell>
+							</ClayTable.Row>
+						</ClayTable.Body>
+					</ClayTable>
+				</TokenItem>
+
+				<TokenItem label="table-title table-list-title" size="large">
+					<Table
+						className="table-title"
+						hover={false}
+						isResponsive={false}
+						text="Spain"
+					/>
+				</TokenItem>
+
+				<TokenItem label="table-valign-bottom" size="large">
```

Comment:
Should be added to the table utilities array.

Reply:
Fixed